release-notes.md

```diff
@@ -352,6 +352,9 @@ This document contains release notes per zib, indicating differences with their
 ## zib-TextResult
 * New concept VisualResult mapped in additional profile zib-TextResult.VisualResult on `Media.content`. `DiagnosticReport.image.link` references this profile.
 
+# zib-TNMTumorClassification
+* There is no previous profile for Education in STU3 and therefore no diff.
```

Comment:
The zib Education is wrongly referenced here.

Reply:
Fix